store/sqlstore/license_store.go

```diff
@@ -50,13 +52,13 @@ func (ls SqlLicenseStore) Save(license *model.LicenseRecord) (*model.LicenseReco
 		Where(sq.Eq{"Id": license.Id})
 	queryString, args, err := query.ToSql()
 	if err != nil {
-		return nil, model.NewAppError("SqlLicenseStore.Save", "store.sql.build_query.app_error", nil, err.Error(), http.StatusInternalServerError)
+		return nil, errors.Wrap(err, "failed to build query to get Licenses")
```

Comment:
I guess this needs to be "license_tosql" ?

Reply:
Ok